singularity_threshold: 60
warning_threshold: 40
ee_tf_name: 'panda_link0'
move_group_name: 'panda_arm'
